Resources/Prototypes/_Moffstation/Entities/Objects/Consumable/Food/Drinks/drinks_bottles.yml

```diff
@@ -0,0 +1,21 @@
+- type: entity
+  parent: [DrinkWaterBottleFull]
+  id: DrinkAmmoniaBottleFull
+  name: ammonia bottle
+  description: A bottle adveritising 100% naturally-sourced ammonia. You think that maybe you dont want to know what the source is.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        reagents:
+        - ReagentId: Ammonia
+          Quantity: 30
+  - type: Sprite
+    sprite: _Moffstation/Objects/Consumable/Drinks/ammoniabottle.rsi
+  - type: Label
+    currentLabel: reagent-name-ammonia
+  - type: Sealable
+  - type: Tag
+    tags:
+    - SurvivalBoxInsertable
+    - DrinkCup
```

Comment:
```suggestion
```
Shouldn't need to redefine the tags (unless you specifically want it to be `DrinkCup` and not `DrinkBottle`, and `Sealable` isn't on the normal water bottle, so it shouldn't be added to this one.